src/compile/selection/selection.ts

```diff
@@ -413,3 +418,14 @@ export function positionalProjections(selCmpt: SelectionComponent) {
   });
   return {x, xi, y, yi};
 }
+
+export function assembleInit(
+  init: SelectionInit | SelectionInitArray,
+  wrap: (str: string) => string = identity
+): string {
+  return isArray(init)
+    ? `[${(init as SelectionInit[]).map(v => assembleInit(v, wrap)).join(', ')}]`
```

Comment:
@arvind I've try to resolve conflicts in this branch with master. 
However, I'm getting some type errors, and it's unclear to me why we should to the casting (`as SelectionInit[]`) on this line.  

Shouldn't we fix the type of the `init` above?

Reply:
Actually, I just realize the Dom is the one who wrote this line.

Reply:
Ok fixed.